src/NuGetPackageConfigConverter/Services/PackageManagerConverter.cs

```diff
@@ -154,16 +214,20 @@ private bool RemovePackages(Project project, IEnumerable<string> ids, Cancellati
 
         private static ProjectItem GetPackageConfig(Project project) => GetProjectItem(project.ProjectItems, "packages.config");
 
-        private void RemoveDependencyFiles(IEnumerable<Project> projects)
+        private void RemoveDependencyFiles(IEnumerable<Project> projects, ConverterUpdateViewModel model)
         {
-            foreach(var project in projects)
+
+            foreach (var project in projects)
             {
+                model.Status = $"Removing dependency files for '{project.Name}'";
+                logger.Log(model.Status);
                 RemoveDependencyFiles(project);
             }
         }
 
         private static void RemoveDependencyFiles(Project project)
         {
+
```

Comment:
nit: there are a number of extra newlines in this file

Reply:
yep